doc/digital-receipt/implementation/getting-started.md

```diff
@@ -0,0 +1,25 @@
+---
+slug: /poscreators/middleware-doc/digital-receipt/getting-started
+title: 'Getting started'
+---
+
+# Getting started
+
+In this section, the implementation steps to achieve and ensure a complete integration into your Point of Sale software are described. 
+
+>[!NOTE]
+>Currently the digital receipt can only be implemented into existing middleware integrations. An own standalone digital receipt API is not available yet. 
+
+This high level overview shows you the implementation and configuration steps, who are required for the digital receipt via QR-Code, Give-Away (QR-Label) and the InStore App. 
+
+<br/>
+
+![Screenshot 2023-11-06 145343](https://github.com/fiskaltrust/interface-doc/assets/124153755/c448df73-f9cd-4dac-bf86-35827e38bb7c)
```

Comment:
Please put the pics into the repo and add proper alt texts instead of "Screenshot" or "Bild" (in the whole file)